crates/bevy_app/src/app.rs

```diff
@@ -73,10 +78,12 @@ impl Default for App {
 }
 
 impl App {
+    /// Creates a new [`App`].
     pub fn new() -> App {
         App::default()
     }
 
+    /// Creates a new empty [`App`].
```

Comment:
What's the practical difference between this and `App::new`?

How do they differ, and why would you want to use this instead? :)